docs/manual/comparison_with_rdatatable.rst

```diff
@@ -0,0 +1,330 @@
+.. py:currentmodule:: datatable
+
+Comparison with Rdatatable
+==========================
+
+``Datatable`` strives to mimic `Rdatatable's <https://rdatatable.gitlab.io/data.table/index.html>`_ API as closely as possible; however, there are differences due to language constraints.
```

Comment:
I would say we strive to implement the same functionality as there is in now in `data.table`, as for the API we normally try to get the best of `data.table`, `pandas`, `numpy`, etc.

Reply:
Changed the description here with text from the readme section of the library